views/components/VirtualTables/ParticipantsTable/index.jsx

```diff
@@ -1,101 +1,76 @@
-      }
-      height={props.height}
-      headerHeight={TABLE_ROW_HEIGHT}
-      headerClassName="ParticipantTableHeader"
-      rowHeight={TABLE_ROW_HEIGHT}
-      rowCount={props.rowCount}
-      rowGetter={({ index }) => props.participants.find((_, i) => i === index)}
-      rowClassName="ParticipantTableRow"
-      sort={props.sort}
-      sortBy={props.sortBy}
-      sortDirection={props.sortDirection}
-    >
-      <Column
-        label="Participant"
-        dataKey="subject"
-        width={props.width / 5}
-        cellRenderer={({ rowData: { study, subject } }) => {
-          return (
-            <Link
-              style={{ textDecoration: 'none' }}
-              to={routes.dashboard(study, subject)}
-            >
-              {subject}
-            </Link>
-          )
-        }}
-      />
-      <Column
-        label="Study"
-        dataKey="study"
-        width={props.width / 5}
-        cellRenderer={({ rowData }) => {
-          return (
-            <Link
-              style={{ textDecoration: 'none' }}
-              to={`/dashboard/${rowData.study}`}
-            >
-              {rowData.study}
-            </Link>
-          )
-        }}
-      />
-      <Column
-        label="Complete"
-        cellRenderer={({ rowData: { study, subject, complete } }) => (
-          <Checkbox
-            name={`complete-${study}`}
-            icon={<CheckBoxOutlineBlankIcon />}
-            checkedIcon={
-              <CheckBoxIcon style={{ color: 'rgba(0, 0, 0, 0.54)' }} />
-            }
-            checked={complete}
-            disableRipple={true}
-            value={subject}
-            onChange={props.onUpdate}
-          />
-        )}
-        width={props.width / 5}
-      />
-      <Column
-        label="Star"
-        cellRenderer={({ rowData: { study, subject, star } }) => (
+  const {
+    onSort,
+    sortProperty,
+    sortDirection,
+    sortable,
+    participants,
+    onUpdate,
+  } = props
+  const handleRequestSort = (_event, property) => {
+    const isAsc =
+      sortProperty === property && sortDirection === SORT_DIRECTION.ASC
+    return onSort(property, isAsc ? SORT_DIRECTION.DESC : SORT_DIRECTION.ASC)
+  }
+  const headers = [
+    {
+      dataProperty: 'subject',
+      label: 'Participant ID',
+      sortable: !!sortable,
+    },
+    {
+      dataProperty: 'study',
+      label: 'Study',
+      sortable: !!sortable,
+    },
+    {
+      dataProperty: 'days',
+      label: 'Days In Study',
+      sortable: !!sortable,
+    },
+    {
+      dataProperty: 'star',
+      label: '',
+      sortable: false,
+    },
+  ]
+  const cellRenderer = (participant, property) => {
+    switch (property) {
+      case 'star':
+        return (
           <Checkbox
-            name={`star-${study}`}
+            name={`star-${participant.study}`}
             disableRipple={true}
             icon={<StarBorder />}
-            checked={star}
+            checked={participant.star || false}
```

Comment:
The backend request should handle this
https://github.com/AMP-SCZ/dpdash/blob/fbeec7fcb3c7eca137e642db7206d848fae94059/server/models/ParticipantsModel/index.js#L31-L35

I think completed will probably have to change